twinkle.js

```diff
@@ -262,11 +262,24 @@ Twinkle.addPortlet = function(navigation, id, text, type, nextnodeid) {
 		}
 		return null;
 	}
+
 	if (type === 'menu') {
 		// In order to get mw.util.addPortlet to generate a dropdown menu in vector and vector-2022, the nextnodeid must be p-cactions. Any other nextnodeid will generate a non-dropdown portlet instead.
 		nextnodeid = 'p-cactions';
 	}
-	return mw.util.addPortlet(id, text, '#' + nextnodeid);
+
+	let portlet = mw.util.addPortlet(id, text, '#' + nextnodeid);
```

Comment:
Should this be var?

Does this code get transpiled before it's deployed to wikis?

I saw you are using let but not template syntax  (`#{nextnodeid}` ) and given the rest of the file uses `var` I'm not sure which syntax this should be written in.

Reply:
We don't transpile. Went ahead and changed to var for consistency.